yarn-project/noir-contracts/src/contracts/noir-aztec3/src/messaging/l1_message.nr

```diff
@@ -0,0 +1,109 @@
+
+use crate::abi::Inputs;
+use crate::types::point::Point;
+use crate::messaging::messaging_getter_data::L1ToL2MessageGetterData;
+use crate::messaging::messaging_getter_data::make_l1_to_l2_message_getter_data;
+use crate::context::PrivateFunctionContext;
+
+// TODO: Still need to check that the secret hash -> note hash
+// this should be another pedersen generator ?
+global L1_MESSAGE_LEN = 8;
```

Comment:
For consistency, think it would be nice if it is referred to as `L1_TO_L2`, generally for this file some places it is L1Message but and other L1ToL2.